File: Mobile App/scratch/fix_colors.py
import os
import re

def fix_with_values(directory):
    pattern = re.compile(r'withValues\(alpha:\s*([^)]+)\)')
    for root, _, files in os.walk(directory):
        for file in files:
            if file.endswith('.dart'):
                path = os.path.join(root, file)
                with open(path, 'r', encoding='utf-8') as f:
                    content = f.read()
                
                new_content = pattern.sub(r'withOpacity(\1)', content)
                
                if new_content != content:
                    with open(path, 'w', encoding='utf-8') as f:
                        f.write(new_content)
                    print(f"Fixed: {path}")

if __name__ == "__main__":
    fix_with_values('lib')

File: Mobile App/scratch/fix_colors.js
const fs = require('fs');
const path = require('path');

function walk(dir) {
    let results = [];
    const list = fs.readdirSync(dir);
    list.forEach(file => {
        file = path.resolve(dir, file);
        const stat = fs.statSync(file);
        if (stat && stat.isDirectory()) {
            results = results.concat(walk(file));
        } else {
            if (file.endsWith('.dart')) results.push(file);
        }
    });
    return results;
}

const files = walk('lib');
files.forEach(file => {
    let content = fs.readFileSync(file, 'utf8');
    const pattern = /withValues\(alpha:\s*([^)]+)\)/g;
    const newContent = content.replace(pattern, 'withOpacity($1)');
    if (content !== newContent) {
        fs.writeFileSync(file, newContent);
        console.log(`Fixed: ${file}`);
    }
});
